include/spirv-tools/libspirv.h

```diff
@@ -370,7 +370,7 @@ typedef enum {
 const char* spvTargetEnvDescription(spv_target_env env);
 
 // Creates a context object.  Returns null if env is invalid.
-spv_context spvContextCreate(spv_target_env env);
+spv_context spvContextCreate(spv_target_env env, bool permissive = false);
```

Comment:
If we need to add a new entry point anyway, I'd prefer to have that new entry point designed as an extensible one. It's hard to predict the future; if we indeed need another entry point in the future, introducing yet another entry point is not optimal, especially if the new variant is orthogonal to the existing options.